File: lib/Screens/calling_modules/models/calling_dashboard_model.dart
class CallingDashboardModel {
  String? status;
  String? message;
  String? lastUpdatedOn;
  List<CallingDashboardOutput>? output;

  CallingDashboardModel({
    this.status,
    this.message,
    this.lastUpdatedOn,
    this.output,
  });

  CallingDashboardModel.fromJson(Map<String, dynamic> json) {
    status = json['status'];
    message = json['message'];
    lastUpdatedOn = json['LastUpdatedOn'];
    if (json['output'] != null) {
      output = <CallingDashboardOutput>[];
      json['output'].forEach((v) {
        output!.add(CallingDashboardOutput.fromJson(v));
      });
    }
  }
}

class CallingDashboardOutput {
  String? columnName;
  String? totalValue;

  CallingDashboardOutput({this.columnName, this.totalValue});

  CallingDashboardOutput.fromJson(Map<String, dynamic> json) {
    columnName = json['ColumnName'];
    totalValue = json['TotalValue']?.toString();
  }
}

File: S2T Operational - After - 9.82/S2T Operational - After - 9.82/app/src/main/java/com/myhindlab/abkat/activities/calling_dashboard/model/CallingDashboardModel.java
package com.myhindlab.abkat.activities.calling_dashboard.model;


import java.util.List;

import javax.annotation.processing.Generated;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class CallingDashboardModel {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("LastUpdatedOn")
    @Expose
    private String lastUpdatedOn;

    public String getLastUpdatedOn() {
        return lastUpdatedOn;
    }

    public void setLastUpdatedOn(String lastUpdatedOn) {
        this.lastUpdatedOn = lastUpdatedOn;
    }

    @SerializedName("output")
    @Expose
    private List<Output> output;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Output> getOutput() {
        return output;
    }

    public void setOutput(List<Output> output) {
        this.output = output;
    }


    @Generated("jsonschema2pojo")
    public class Output {

        @SerializedName("ColumnName")
        @Expose
        private String columnName;
        @SerializedName("TotalValue")
        @Expose
        private String value;



        public String getColumnName() {
            return columnName;
        }

        public void setColumnName(String columnName) {
            this.columnName = columnName;
        }

        public String getValue() {
            return value;
        }

        public void setValue(String value) {
            this.value = value;
        }

    }
}